app/io/pathfinder/controllers.scala

```diff
@@ -0,0 +1,98 @@
+package io.pathfinder
```

Comment:
Is there any reason that this needs to be in one file?

For consistency, I think we should define each controller in it's own file. Having a file called `controllers.scala` implies that all of our controllers are in it.